tests/test_main.py

```diff
@@ -1374,3 +1376,47 @@ class M(BaseModel):
         a: int
 
     get_type_hints(M.__config__)
+
+
+def foo(arg1, arg2):
```

Comment:
could we move this out of `test_main.py` e.g. into `test_dataclasses.py` or `test_edge_cases.py`?

Reply:
Moved to `test_dataclasses` with model creations inside the test